src/main/java/uk/nhs/digital/nhsconnect/nhais/service/TimestampService.java

```diff
@@ -1,15 +1,23 @@
 package uk.nhs.digital.nhsconnect.nhais.service;
 
 import org.springframework.stereotype.Component;
-import uk.nhs.digital.nhsconnect.nhais.utils.TimestampUtils;
 
-import java.time.ZonedDateTime;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
 
 @Component
 public class TimestampService {
+    public static final ZoneId UKZone = ZoneId.of("Europe/London");
 
-    public ZonedDateTime getCurrentTimestamp() {
-        return TimestampUtils.getCurrentDateTimeAsUTC();
+    public Instant getCurrentTimestamp() {
+        var now = Instant.now();
+        return now.minusNanos(now.getNano());
     }
 
+    public String formatInISO(Instant timestamp) {
+        return DateTimeFormatter.ISO_DATE_TIME
+            .withZone(TimestampService.UKZone)
```

Comment:
Could we keep these UTC instead of UK local time? I'll make a Confluence update to specify what sorts of times we use where.

Reply:
In tests it has to be UTC to see that the formatting with UK zone works correctly.
If I would set UK time here, it would always translate to the same value